src/Helpers/Form/Utils.php

```diff
@@ -83,3 +82,46 @@ function getFailedTransactionPageURL( $args = array() ) {
 	);
 }
 
+/**
+ * Get shortcode argument.
+ *
+ * @since 2.7.0
+ */
+function getShortcodeArgs() {
+	if ( ! isViewingForm() ) {
+		return false;
+	}
+
+	$queryString = array_map( 'give_clean', wp_parse_args( $_SERVER['QUERY_STRING'] ) );
+
+	return array_intersect_key( $queryString, give_get_default_form_shortcode_args() );
+}
+
+/**
+ * Get form ID.
```

Comment:
Same thing here. When would someone reach for this utility?

Reply:
fixed: https://github.com/impress-org/givewp/compare/15fa2b13310e...b38638d7c2b3